src/utils/constants.js

```diff
@@ -5,6 +5,12 @@ export const OBJECT_NAMES = {
   PERMISSIONS_REQUIRED_SIGNOFF: 'required_signoffs/permissions',
   PRODUCT_REQUIRED_SIGNOFF: 'required_signoffs/product',
 };
+export const GCS_CONFIG = {
+  NIGHTLY_HISTORY_BUCKET:
+    'https://www.googleapis.com/storage/v1/b/balrog-prod-nightly-history-v1/o',
```

Comment:
This is going to differ depending on the environment we're deployed in. Can we read it from the environment? These are the correct URLs for local dev, so they can go in `.neutrinorc.js`.